config/config.go

```diff
@@ -217,6 +218,13 @@ func (c *Configuration) copyOverWithLog(overrides Log) {
 	c.Log.SyslogEndpoint = utils.Coalesce(overrides.SyslogEndpoint, c.Log.SyslogEndpoint).(string)
 }
 
+// --- JWT.
+func (c *Configuration) copyOverWithJwt(overrides Jwt) {
+	c.Jwt.Included_paths = utils.Coalesce(overrides.Included_paths, c.Jwt.Included_paths).([]string)
+	c.Jwt.Allowed_scopes = utils.Coalesce(overrides.Allowed_scopes, c.Jwt.Allowed_scopes).([]string)
+	c.Jwt.Jwks_url = utils.Coalesce(overrides.Jwks_url, c.Jwt.Jwks_url).(string)
```

Comment:
I'd change the following to be consistent with the naming convention of all other variables:

```suggestion
	c.Jwt.IncludedPaths = utils.Coalesce(overrides.IncludedPaths, c.Jwt.IncludedPaths).([]string)
	c.Jwt.AllowedScopes = utils.Coalesce(overrides.AllowedScopes, c.Jwt.AllowedScopes).([]string)
	c.Jwt.JwksUrl = utils.Coalesce(overrides.JwksUrl, c.Jwt.JwksUrl).(string)
```

Reply:
done!